web/src/components/FilingsCalendarAsList.tsx

```diff
@@ -28,18 +33,30 @@ export const FilingsCalendarAsList = (props: Props): ReactElement => {
             key={`${filing.identifier}-${filing.dueDate}`}
             className="margin-top-6 bg-roadmap-blue border-base-lightest border-2px padding-top-3 padding-bottom-1 padding-x-4 radius-lg"
           >
-            <div className="flex">
-              <div className="h3-styling text-normal">{Config.dashboardDefaults.calendarHeader}</div>
-              <div>
-                <ArrowTooltip title={Config.dashboardDefaults.calendarTooltip}>
-                  <div
-                    className="fdr fac margin-left-1 margin-bottom-05 font-body-lg text-green"
-                    data-testid="calendar-tooltip"
-                  >
-                    <Icon>help_outline</Icon>
-                  </div>
-                </ArrowTooltip>
+            <div className="flex flex-align-end padding-top-2 margin-bottom-1 flex-justify">
+              <div className="flex flex-align-end">
+                <h2 className="margin-bottom-0">{Config.dashboardDefaults.calendarHeader}</h2>
+                <div className="margin-top-05">
+                  <ArrowTooltip title={Config.dashboardDefaults.calendarTooltip}>
+                    <div
+                      className="fdr fac margin-left-1 margin-bottom-05 font-body-lg text-green"
+                      data-testid="calendar-tooltip"
+                    >
+                      <Icon>help_outline</Icon>
+                    </div>
+                  </ArrowTooltip>
+                </div>
               </div>
+              {isDesktopAndUp && (
+                <Button
+                  style="light"
+                  className="font-body-2xs padding-y-05 padding-x-1"
+                  onClick={props.onToggle}
+                >
+                  <Icon className="usa-icon--size-3 margin-right-05">menu</Icon>
+                  {Config.dashboardDefaults.calendarFullViewButton}
+                </Button>
+              )}
```

Comment:
I think there's a bug here - I don't think this button should be visible on the list view when the onToggle function doesn't exist. Currently, it shows up in the list-view calendar after a user enters their formation date 
![image](https://user-images.githubusercontent.com/6563663/184655508-ad066d27-3523-45cf-8f60-d50c46bcb102.png)

Reply:
The double header is also back... The first Upcoming header should be removed

Reply:
I figured these were related...but yeah it looks like the rendering is off somehow

Reply:
1. I am not able to duplicate the double header behavior. 
2. I updated the button logic.
Thanks!